tests/dummy/app/styles/app.scss

```diff
@@ -104,7 +98,7 @@ a {
     h1,
     p {
       @include font-smoothing;
-      color: $c-text-color;
+      color: _e3dn('text-color');
```

Comment:
If we agree, I'll go through and adjust the config settings and then when they are applied to only touch components added by the addon. I'll then re-add to the dummy app the right body bg and text color.